jmxtrans-cluster/src/main/java/com/googlecode/jmxtrans/cluster/ClusterManager.java

```diff
@@ -0,0 +1,48 @@
+package com.googlecode.jmxtrans.cluster;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.inject.Guice;
+import com.google.inject.Inject;
+import com.google.inject.Injector;
+import org.apache.commons.configuration.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+
+/**
+ * ClusterManager Tester.
+ *
+ * @author Tibor Kulcsar
+ * @version 1.0
```

Comment:
Could you either remove the `@version` or align them with the actual version of JmxTrans?